composer.json

```diff
@@ -152,7 +161,10 @@
         "drupal/search_api": "^1.0",
         "drupal/jsonapi": "^1.1",
         "drupal/openapi": "1.0-alpha1",
-        "drupal/simple_oauth": "^2.0@RC"
+        "drupal/simple_oauth": "^2.0@RC",
+        "oomphinc/composer-installers-extender": "^1.1",
+        "drupal/dropzonejs": "^1.0@alpha",
```

Comment:
We should get rid of the superfluous @ stability tags too. If we're going to assume the scaffold project has the proper installer paths, we can assume it has stability set to dev too.